lib/block_view_dao_coin_limit_order.go

```diff
@@ -1029,6 +936,259 @@ func (bav *UtxoView) _deleteDAOCoinLimitOrderEntryMappings(entry *DAOCoinLimitOr
 	bav._setDAOCoinLimitOrderEntryMappings(&tombstoneEntry)
 }
 
+func _calculateDAOCoinsTransferredInLimitOrderMatch(
+	transactorOrder *DAOCoinLimitOrderEntry,
+	matchingOrder *DAOCoinLimitOrderEntry,
+	transactorQuantityToFillInBaseUnits *uint256.Int) (
+	updatedTransactorQuantityToFillInBaseUnits *uint256.Int,
+	updatedMatchingQuantityToFillInBaseUnits *uint256.Int,
+	transactorBuyingCoinBaseUnitsTransferred *uint256.Int,
+	transactorSellingCoinBaseUnitsTransferred *uint256.Int,
+	err error) {
```

Comment:
```suggestion
	_updatedTransactorQuantityToFillInBaseUnits *uint256.Int,
	_updatedMatchingQuantityToFillInBaseUnits *uint256.Int,
	_transactorBuyingCoinBaseUnitsTransferred *uint256.Int,
	_transactorSellingCoinBaseUnitsTransferred *uint256.Int,
	_err error) {
```
so we explicitly return all values